### ISSUE TITLE
Note twice in graph, local graph view disabled, "shortest-path"-linking corrupted, window-menue shows note twice

### ISSUE THREAD

**Issue by chrizzzlybear** [ras0q/obsidian-graph-banner#64]
![image](https://github.com/user-attachments/assets/41d89219-bda1-4bd3-a1f5-4976a8118154)

A single note (that **really** has no "twin" in the vault) shows twice in graph while the native graph view of Obsidian is disabled.

Furthermore it corrupts file-link "shortest path" linking. This means it gives obsidian the impression that a note, that in fact only exists once in the vault with the single title it has, would exist twice. Accordingly OB shows the behavior it would show if two notes of the same title would exist twice within the vault: linking to the note with the whole path name, setting the title with the pipe character. But it does this without two notes of the same name.

And when you use the file-menue in OB to change from one window to another one, then you see that the current and another window would have the name of note you depart from.

So the questions seems to be, why OB "thinks" it would have two notes of the same name... 😉

By the way: I saw the current behavior with the linking issue only since insider built 1.7.0. but the rest also appeared within 1.6.9.

Good luck with this one - I really like your plugin so hopefully it works again soon.

**Comment by ras0q** [comment]
@chrizzzlybear This plugin simply reuse the HTML Element in the local graph, so I think this is a bug of Obsidian, not this plugin.

**Comment by chrizzzlybear** [comment]
Hm,... possible. At the moment some member of the OB-Team commented on this and said he's quite sure it must be some plugin... The thing is, I can't replicate the error and I don't know how many notes have been changed... and it also happened after I disabled graph view. So at least, I think it is not your plugin... The question would be: What can make your plugin show a note that clearly only exists once, twice? 😉
